Heat_method_3/doc/Heat_method_3/Heat_method_3.txt

```diff
@@ -0,0 +1,249 @@
+
+The sum is taken over all of the neighboring vertices
+\f$j\f$. Further, \f$\alpha_{ij}\f$ and \f$\beta_{ij}\f$ are the
+angles opposing the corresponding edge \f$ij\f$. We express this
+operation via a matrix \f$L = M^{-1}L_c\f$ where \f$M \in
+R^{|V|x|V|}\f$ is a diagonal matrix containing the vertex areas and
+\f$L_c \in R^{|V|x|V|} \f$ is the cotan operator representing the
+remaining sum.
+
+From this, the symmetric positive-definite system
+\f$(M-tL_C)u = \delta_{\gamma}\f$ can be solved to find
+\f$u\f$ where \f$\delta_{\gamma}\f$ is the Kronecker delta over \f$\gamma\f$.
+
+Next, the gradient in a given triangle can be expressed as:
+
+\f$\nabla \cdot X = \frac{1}{2} \sum_j cot\theta_1 (e_1 \cdot X_j) + cot \theta_2 (e_2 \cdot X_j)\f$
+
+where the sum is taken over incident triangles \f$j\f$ each with a vector \f$X_j\f$,
+\f$e_1\f$ and \f$e_2\f$ are the two edge vectors of triangle \f$j\f$
+containing \f$i\f$ and \f$\theta_1\f$, \f$\theta_2\f$ are the opposing angles.
+
+Finally, let \f$b \in R^{|V|}\f$ be the integrated divergences of the normalized vector field X.
+Thus, solving the symmetric Poisson problem \f$ L_c \phi = b\f$ computes the final distance function.
+
+\subsection Subsection_HM_IDT_Definitions Intrinsic Delaunay Triangulation
+
+The standard discretization of the cotan Laplace operator uses the cotangents of the angles in the triangle mesh.
+The intrinsic Delaunay algorithm constructs an alternative triangulation of the same polyhedral surface, which
+in turn yields a different (typically more accurate) cotan Laplace operator.  Conceptually, the edges of the iDT
+still connect pairs of vertices from the original (input) surface, but are now allowed to be geodesic paths along
+the polyhedron and do not have to correspond to edges of the input triangulation.  These paths are not stored
+explicitly; instead, we simply keep track of their lengths as the triangulation is updated.  These lengths are
+sufficient to determine areas and angles of the intrinsic triangles, and in turn, the new cotan Laplace matrix.
+
+An edge of a mesh is locally Delaunay if the sum of opposite angles is no smaller than pi, or equivalently,
+if the cotangents of the opposing angles is nonnegative. A mesh is Delaunay if all of its edges are locally Delaunay.
+
+A standard algorithm to convert a given planar triangulation into a Delaunay triangulation is
+to flip non-Delaunay edges in a mesh until the mesh is Delaunay.
+Similarly, the intrinsic Delaunay triangulation of a simplicial surface
+is constructed by performing intrinsic edge flips.
+
+Let \f$ K = (V,E,T) \f$ be a 2-manifold triangle mesh, where \f$V\f$ is the vertex set,
+\f$ E \f$ is the edge set and \f$ T \f$ is the face set (triangle set).
+Let \f$ L \f$ be the set of Euclidean distances, where \f$ L(e_{ij}) = l_{ij} = || p_i - p_j || \f$ ,
```

Comment:
L seems to be used for another purpose in the paragraph above (laplacien AFAIU). It might be confusing for the user.

Reply:
Here you can replace L by D